tests/test_tng.py

```diff
@@ -283,42 +315,49 @@ def test_water_npt_uncompressed_vels_forces_first_vels(
     WATER_NPT_UNCOMPRESSED_VELS_FORCES, WATER_NPT_UNCOMPRESSED_VELS_FORCES_DATA
 ):
     with pytng.TNGFileIterator(WATER_NPT_UNCOMPRESSED_VELS_FORCES) as tng:
-        first_frame_first_10_vels = tng[0].vel[:10, :]
+        vel = np.zeros((2700, 3), dtype=np.float32)
+        vel = tng[0].get_velocities(vel)
         assert_array_almost_equal(
             WATER_NPT_UNCOMPRESSED_VELS_FORCES_DATA.first_frame_first_10_vels,
-            first_frame_first_10_vels,
+            vel[:10, :],
             decimal=2,
         )  # decimal = 2 really slack
 
+
 def test_water_npt_uncompressed_vels_forces_last_vels(
     WATER_NPT_UNCOMPRESSED_VELS_FORCES, WATER_NPT_UNCOMPRESSED_VELS_FORCES_DATA
 ):
     with pytng.TNGFileIterator(WATER_NPT_UNCOMPRESSED_VELS_FORCES) as tng:
-        last_frame_last_10_vels = tng[len(tng)-1].vel[2690:2700, : ]
+        vel = np.zeros((2700, 3), dtype=np.float32)
+        vel = tng[len(tng)-1].get_velocities(vel)
         assert_array_almost_equal(
             WATER_NPT_UNCOMPRESSED_VELS_FORCES_DATA.last_frame_last_10_vels,
-            last_frame_last_10_vels,
-            decimal=2,
+            vel[2690:2700, :],
+            decimal=2
         )  # decimal = 2 really slack
 
+
 def test_water_npt_uncompressed_vels_forces_first_frc(
     WATER_NPT_UNCOMPRESSED_VELS_FORCES, WATER_NPT_UNCOMPRESSED_VELS_FORCES_DATA
 ):
     with pytng.TNGFileIterator(WATER_NPT_UNCOMPRESSED_VELS_FORCES) as tng:
-        first_frame_first_10_frc = tng[0].frc[:10, :]  # todo forces
+        frc = np.zeros((2700, 3), dtype=np.float32)
+        frc = tng[0].get_forces(frc)  # todo forces
         assert_array_almost_equal(
             WATER_NPT_UNCOMPRESSED_VELS_FORCES_DATA.first_frame_first_10_frc,
-            first_frame_first_10_frc,
+            frc[:10, :],
             decimal=2,
         )  # decimal = 2 really slack
 
+
 def test_water_npt_uncompressed_vels_forces_last_frc(
     WATER_NPT_UNCOMPRESSED_VELS_FORCES, WATER_NPT_UNCOMPRESSED_VELS_FORCES_DATA
 ):
     with pytng.TNGFileIterator(WATER_NPT_UNCOMPRESSED_VELS_FORCES) as tng:
-        last_frame_last_10_frc = tng[len(tng) -1].frc[2690:2700, :]
+        frc = np.zeros((2700, 3), dtype=np.float32)
```

Comment:
Is there a test for handling "invalid" `dtype` for the array used to store the data? If you feed in `np.complex128` or a unicode type or something?